CHANGELOG.md

```diff
@@ -19,7 +19,7 @@ All notable changes to this project will be documented in this file. See [standa
 
 ### Bug Fixes
 
-* add flag to allow control of intrumenter esModules option, default to looser parsing ([#863](https://github.com/istanbuljs/nyc/issues/863)) ([6b6cd5e](https://github.com/istanbuljs/nyc/commit/6b6cd5e))
+* add flag to allow control of instrumenter esModules option, default to looser parsing ([#863](https://github.com/istanbuljs/nyc/issues/863)) ([6b6cd5e](https://github.com/istanbuljs/nyc/commit/6b6cd5e))
```

Comment:
@bcoe does this create a problem with the utility which generates the CHANGELOG.md?

Really the problem is that the typo is in the commit log, I'm not sure this can be fixed?

Reply:
Sorry, didn't realize that would be a problem and think to look if it was generated, I was on typo autopilot there.  
If it's just going to disappear next time the tool runs then we'll remove it

Reply:
the changelog generation looks at the header, doesn't re-generate the whole document, so the edit should be fine.